client-cli/src/main/java/au/org/consumerdatastandards/client/cli/ReferenceTest.java

```diff
@@ -56,22 +56,20 @@ public void validatePath(@ShellOption(value = "-f", help = "payload file or fold
         } else if (file.isDirectory()) {
             File[] files = file.listFiles();
             for (File oneFile : files) {
-                if (StringUtils.isBlank(model)) {
-                    payloadValidator.validateFile(new File(oneFile.getAbsolutePath()));
-                } else {
-                    payloadValidator.validateFile(new File(oneFile.getAbsolutePath()), model);
-                }
+                validatePath(oneFile.getAbsolutePath(), model);
             }
         } else {
-            List<ConformanceError> payloadErrors = null;
+            List<ConformanceError> payloadErrors;
             if (StringUtils.isBlank(model)) {
                 payloadErrors = payloadValidator.validateFile(file);
             } else {
                 payloadErrors = payloadValidator.validateFile(file, model);
             }
```

Comment:
Can (not _has to_) be replaced with `List<ConformanceError> payloadErrors = (StringUtils.isBlank(model) ? payloadValidator.validateFile(file) : payloadValidator.validateFile(file, model));`

Reply:
ok, updated accordingly